tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -1536,10 +1542,11 @@ def test_task_client_error_3_times(
 
         event["input"] = {"granules": [gran]}
         mock_s3_cli = mock_boto3_client("s3")
+
         mock_s3_cli.restore_object.side_effect = [
-            None,
+            {"ResponseMetadata": {"HTTPStatusCode": 200}},
```

Comment:
The status value should be 202 on an initial restore. 200 occurs if the granule has already been restored.

Reply:
Fixed